crates/flashblocks-rpc/src/state.rs

```diff
@@ -280,9 +291,16 @@ where
 
             gas_used = receipt.cumulative_gas_used();
             next_log_index += receipt.logs().len();
-
+        }
+        let mut state_cache_builder = StateOverridesBuilder::default();
+        // Execute recovered transaction that belongs to the last flashblocks
+        for tx in recovered_transactions
+            .into_iter()
+            .rev()
+            .take(latest_flashblock.diff.transactions.len())
```

Comment:
Is this going to execute them back to front for the last flashblock? i.e. if the txns in order are:

fb1-tx1, fb1-tx2, fb2-tx3, fb2-tx4

Will this execute tx4, then tx3 when it should be the other way around? I think we may need to do rev/take/rev

Reply:
Ohhh, my bad, though im smart by doing this